templates/releases/header.html

```diff
@@ -9,8 +9,7 @@
         * `failures`
         * `activity`
         * `queue`
-        * `author`
-    * `author` A string, used for the authors page
+        * `author` A string, used for the authors page
 #}
 {% macro header(title, description, tab, author=false) %}
```

Comment:
Is this ever used?

Reply:
I think it's still working for owner.

Reply:
![image](https://user-images.githubusercontent.com/29879298/111796033-ba17e480-8902-11eb-816f-a7adba3bdba5.png)

Reply:
Great, could you rename it from "author" to "owner" then?

Reply:
> Great, could you rename it from "author" to "owner" then?

Maybe can do it in https://github.com/rust-lang/docs.rs/issues/1323.